src/Phinx/Migration/AbstractMigration.php

```diff
@@ -335,4 +345,24 @@ public function preFlightCheck($direction = null)
             }
         }
     }
+
+    /**
+     * Perform checks on the migration after completion
+     *
+     * Right now, the only check is whether all changes were committed
+     *
+     * @param string|null $direction
+     *
+     * @return void
+     */
+    public function postFlightCheck($direction = null)
+    {
+        foreach ($this->tables as $table) {
+            if ($table->hasPendingActions()) {
+                throw new \RuntimeException('Migration has pending actions after execution!');
+            }
+        }
+
+        return true;
```

Comment:
Why this one?

Reply:
Mainly to make sure there is something to test and I was thinking about possible future change to not throw and instead return false, but I guess it makes no sense to think about that yet.

How should I make sure the test is sensible if the only action is to not throw an exception?

Reply:
It either fails with that exception or not. I would say.